### PR TITLE
First pass through this to switch from ASC

### PR DESCRIPTION
**Pull request by rbeyer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,26 +64,25 @@ bodies such as the Working Groups.
 
 TSC members are responsible for top level technical community concerns. The role
 is mostly administrative and is responsible for admitting new Top Level
-Projects, Top Level Working Groups, and advocating for any needs in the
-technical side of the foundation to the ASC Management Team.
+Projects, Top Level Working Groups..
```
Comment (jlaura) [inline README.md]: double '..'

File: README.md [hunk 2]
```diff
@@ -1,53 +1,29 @@
...
+The TSC exercises autonomy in managing its responsibilities.
 
-### ASC Core
 
-*ASC Core* is defined as the contents of the repository located at
-<https://github.com/USGS-Astrogeology/ISIS3>. This is the location of the ISIS3 low-level API and applications. As the ASC continues to modularize ISIS3 and explores potential name changes this document will be updated appropriately.
+### Planetary Software projects under TSC Stewardship
 
-### Dependent Projects
+None (maybe planetarypy soon?)
```
Comment (jlaura) [inline README.md]: Maybe just omit for now?

**Review by jlaura — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,53 +1,29 @@
-# The ASC Software Organization Technical Steering Committee
+# The Planetary Software Organization Technical Steering Committee
 
-The ASC Software Organization Technical Steering Committee (TSC) is the technical
-governing body of the ASC Software Organization. It is described in the
+The Planeatry Software Organization Technical Steering Committee (TSC) is the technical
```
Comment (michaelaye) [inline README.md]: Planetary

Comment (michaelaye) [reply README.md]: Wondering if I could actually be more helpful here and submit a correction myself? Is that easily possible. I guess I would have to check out your patch-3 branch, would I have write rights on that? I guess write rights aren't branch-dependent so this should work... trying it out...

**Comment by michaelaye** [comment]
So, I don't understand this. I pushed a typo fix (d84a3e0c264934b90002cae2597aaec37950e9db) to rbeyer-patch-3, why doesn't appear here?

File: README.md (hunk 1 above)
Comment (rbeyer) [reply README.md]: Added some text.

File: README.md (hunk 2 above)
Comment (rbeyer) [reply README.md]: Edited.

File: README.md (hunk 3 above)
Comment (rbeyer) [reply README.md]: That's a great question, that I don't know the answer to.  For these, I've just been using the GitHub web pages for all of the edits I'm doing.  I don't have this repo (or the patches) cloned on my machine.  Maybe I'm doing something wrong?  Totally possible.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**